system-test/datastore.ts

```diff
@@ -48,8 +48,10 @@ describe('Datastore', () => {
   it('should allocate IDs', done => {
     datastore.allocateIds(datastore.key('Kind'), 10, (err, keys) => {
       assert.ifError(err);
-      assert.strictEqual(keys.length, 10);
-      assert.strictEqual(entity.isKeyComplete(keys[0]), true);
+      if (keys !== null) {
```

Comment:
Wouldn't this be an error if `keys` was null?  Shouldn't this be `assert.ok(keys);`?

Reply:
Yup! That fixed it!